CHANGES_NEXT_RELEASE

```diff
@@ -1 +1,8 @@
-- Fix: support to lazy attributes and commands based in NGSIv2 (#104)
+- Set Nodejs 8 as minimum version in packages.json (effectively removing Nodev6 from supported versions)
```

Comment:
Not sure where this line comes, but it seems not related with the PR. Note that packages.json already uses Nodejs 8 in [last release](https://github.com/telefonicaid/lightweightm2m-iotagent/blob/1.2.0/package.json) so it's not correct include it in the CNR.

Reply:
Fixed c5be33f